matcha-app/Matcha
# Added viewgroup support

### What does this accomplish?
Resolves https://github.com/matcha-app/Matcha/issues/13, https://github.com/matcha-app/Matcha/issues/14
### What is the scope of this change?
Implements the `ViewGroup` and `LinearLayout` into the matcha system

### How can I test this?
```
{
	"type": "LinearLayout",
	"layoutWidth": "match_parent",
	"layoutHeight": "match_parent",
	"backgroundColor": "#efefef",
	"orientation": "vertical",
	"children": [{
		"type": "TextView",
		"text": "Matcha Dynamic Rendering",
		"textSize": "24",
		"textAlignment":"viewStart",
		"paddingHorizontal":"40",
		"paddingVertical":"40",
		"textColor":"#ffffff",
		"backgroundColor":"#0091ea",
		"layoutWidth": "match_parent",
		"layoutHeight": "wrap_content",
		"elevation":"4"
	},{
		type:"LinearLayout",
		padding:"20",
		backgroundColor:"#efefef",
		orientation:"vertical",
		children:[ {
			"type": "TextView",
			"text": "Look here, now there. These native views are rendered dynamically.",
			"textSize": "16",
			"layoutWidth": "match_parent",
			"layoutHeight": "wrap_content",
			"padding":"20"
		}, {
			"type": "TextView",
			"padding":"20",
			"text": "Matcha is dynamic, so when you need to customize your components it's easy. Add some hooks and now you're free to customize them as you see fit.",
			"textSize": "16",
			"layoutWidth": "match_parent",
			"layoutHeight": "wrap_content"
		}]
	}]
}
```

- [x] This build compiles
- [x] I have tested this and AFAIK these changes do not add any additional crashes/bugs
